stock_inventory_hierarchical/README.rst

```diff
@@ -0,0 +1,121 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================================
+Hierarchical structure for Inventory adjustments
+================================================
+
+This module adds a parent-child relationship between inventory adjustments, to
+help users manage complex warehouses.
+Using several inventories, you can distribute the counting to several persons
+and still keep a clear overview of global inventory's status.
+
+The main inventory will be for a general Location, and the sub-inventories
+should contain sub-Locations.
+
+Odoo will make sure the status of the Inventory and it's Sub-Inventories are
+consistent:
+
+* sub-inventories will be prepared automatically when you prepare the main
+  inventory
+* you must finish the sub-inventories before you can finish the main inventory
+* canceling a sub-inventory is only possible is the parent inventories are
```

Comment:
should not it be the other way around?
you can only cancel a parent inventory if all sub-inventories are cancelled?